openpype/lib/plugin_tools.py

```diff
@@ -531,12 +531,20 @@ def should_decompress(file_url):
                 and we can decompress (oiiotool supported)
     """
     if oiio_supported():
-        output = run_subprocess([
-            get_oiio_tools_path(),
-            "--info", "-v", file_url])
-        return "compression: \"dwaa\"" in output or \
-            "compression: \"dwab\"" in output
-
+        try:
```

Comment:
@iLLiCiTiT or @antirotor I have add this here but still not sure if this is the correct way to do it.